core_lib/src/tool/polylinetool.h

```diff
@@ -51,6 +52,7 @@ class PolylineTool : public StrokeTool
 
 private:
     QList<QPointF> mPoints;
+    bool mClosedOverride = false;
```

Comment:
I'd prefer if we could make the naming similar to the calls, like "mClosedPathOverrideEnabled"